dist/redhat/scylla.spec.mustache

```diff
@@ -17,6 +17,9 @@ Obsoletes:	scylla-server < 1.1
 
 %undefine _find_debuginfo_dwz_opts
 
+%undefine _unique_build_ids
+%global _no_recompute_build_ids 1
+
```

Comment:
Please add comments here and an explanation in the changelog.

Reply:
fixed